docs/developing.md

```diff
@@ -57,3 +57,47 @@ $ cd <application project directory>
 $ yalc remove @hypothesis/frontend-shared
 $ yarn --check-files
 ```
+
+## Adding new components
+
+Before adding a component, determine what category of component it will be — simple, presentational or composite (details can be found in the Pattern Library) — and what group of components it belongs with (e.g. Layout, Data Display).
```

Comment:
It would be useful to indicate which page explains the distinction between the component types. You are referring to http://localhost:4001/using-components, right?